tests/data/vrp_ce12804-withvrf.json

```diff
@@ -24928,9 +24928,9 @@
                     "bgpPeerIdentifier": "192.168.189.96",
                     "bgpPeerRemoteAs": 64512,
                     "bgpPeerState": "established",
-                    "bgpPeerAdminStatus": "stop",
-                    "bgpPeerLastErrorCode": null,
-                    "bgpPeerLastErrorSubCode": null,
+                    "bgpPeerAdminStatus": "",
```

Comment:
do we want this to be empty strings or null?

Reply:
This file does not include the OID we need for this AdminStatus. I'd need to collect again snmp with current code.  Cannot do it currently, I don't have a lab device available, and handcleaning prod data is very time consuming (and usually result in a "false" file breaking tests ...)

Reply:
Hum, the value is correctly populated during poll, I need to do some tests with a real device to clarify this. But I don't have an easy way to run this in prod currently (too many patches to apply over my prod instance running stable).

Reply:
@PipoCanaja best way to test is to temporarily add the device to your development install (assuming it can reach the device)... then remove the device after testing.

Reply:
This test file is behaving funny. Testing on my local machine (with DBTEST=1, on a freshly cleaned test DB) gives a result which is not the same as the tests here on github ... drives me crazy. I even tried to recreate it all together, same behaviour.

Reply:
@PipoCanaja did you see your highlighted change is not in the test files anymore :)